docker/deployment/scripts/kubeSyncMongo.sh

```diff
@@ -1,41 +1,39 @@
 #!/bin/bash
 
 set -e
+set -x
 
-echo Mongo Dump
-ssh mongo-xf mongodump
+PRODCONTEXT=--context='languageforge'
+QACONTEXT=--context='qa-languageforge'
+PRODSERVER=$(kubectl $PRODCONTEXT get pods --selector='app=db' -o name | sed -e s'/pod\///')
+QASERVER=$(kubectl $QACONTEXT get pods --selector='app=db' -o name | sed -e s'/pod\///')
 
-echo Remove Scripture Forge data from dump
-ssh mongo-xf rm -r dump/xforge
+echo "Mongo Dump on production"
+kubectl $PRODCONTEXT exec -c db $PRODSERVER -- bash -c "cd /data/db && mongodump"
 
-echo Create tar file of dump
-ssh mongo-xf tar -czvf mongodump.tgz dump
+echo "Create tar file of dump"
+kubectl $PRODCONTEXT exec -c db $PRODSERVER -- bash -c "cd /data/db && tar -czvf mongodump.tgz dump"
 
-echo Copy tar file to local filesystem
-scp mongo-xf:mongodump.tgz .
+echo "Copy tgz file to local filesystem"
+kubectl $PRODCONTEXT cp -c db $PRODSERVER:/data/db/mongodump.tgz .
 
-MONGOPOD=$(kubectl get pods --selector='app=db' -o name | sed -e s'/pod\///')
+echo "See how big mongo dump file is"
+ls -lh mongodump.tgz
 
-echo Copying mongodump.tgz into $MONGOPOD/data/db
-kubectl cp -c db mongodump.tgz $MONGOPOD:/data/db
+echo "Copy tgz file to QA"
```

Comment:
we might need a flag in here to "force" an overwrite in case a previous file by that name already exists